.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
 sudo: false
 
+## Cache composer bits
+cache:
+  directories:
+    - $HOME/.composer/cache
 php:
-  - 5.5
   - 5.6
   - 7.0
 
 before_script:
-  - cd tests/
+  - composer install --prefer-dist
```

Comment:
I know phpunit is on the server, but this way we know which version of phpunit we're using